### PR TITLE
Support parameters

### PR DESCRIPTION
**Pull request by morrislaptop**
See tests for sample use

### CODE REVIEW

**Comment by morrislaptop** [comment]
* bump?

**Comment by Marak** [comment]
@morrislaptop 

This is really nice. I was just starting to implement this myself.

Due to the cross-cutting nature of this change. I will perform an additional review before merging.

Thank you for your continued patience.

File: lib/fake.js [hunk 1]
```diff
@@ -49,7 +57,8 @@ function Fake (faker) {
     var fn = faker[parts[0]][parts[1]];
 
     // replace the found tag with the returned fake value
-    res = str.replace('{{' + method + '}}', fn());
+    eval('var result = fn(' + parameters + ');');
```
Comment (Marak) [inline lib/fake.js]: Use of `eval()` is not safe here.

An error in parameter parsing or invalid parameters could cause un-intended consequences to other scopes of the application.

I went ahead and replaced this with a combination of `JSON.parse` and `fn.call`

All tests should be passing.

**Comment by Marak** [comment]
Should be resolved in `master` now.

Thanks @morrislaptop

**Pull request closed without merging**